src/igmpproxy.h

```diff
@@ -36,7 +36,9 @@
 */
 
 #include "config.h"
-#include "os.h"
+#ifndef __FreeBSD__
+    #include "os.h"
+#endif
```

Comment:
And this is suspicious too. `os.h` is file prepared by autoconf and contains system dependent definitions. It really should be included as soon as possible, therefore after `config.h`. All platform dependent stuff should be done in this `os.h` file and not in platform-independent `igmpproxy.h` file. This is reason why `os.h` exists.

Reply:
ok, I will remove it from the pull request. It can probably be fixed by having types.h, netinet/in.h in os.h etc.